nano/core_test/network.cpp

```diff
@@ -631,48 +631,52 @@ TEST (tcp_listener, tcp_listener_timeout_node_id_handshake)
 #ifndef _WIN32
 TEST (network, peer_max_tcp_attempts)
 {
+	nano::test::system system;
+
 	// Add nodes that can accept TCP connection, but not node ID handshake
 	nano::node_flags node_flags;
 	node_flags.disable_connection_cleanup = true;
-	nano::test::system system;
-	auto node = system.add_node (node_flags);
-	for (auto i (0); i < node->network_params.network.max_peers_per_ip; ++i)
+	nano::node_config node_config = system.default_config ();
+	node_config.network.max_peers_per_ip = 3;
```

Comment:
This test used to blow up within TSAN, is this changed to address that?

Reply:
It's the same reason as for `max_connections_per_ip` test described below. IO is running continuously now, it's impractical to simulate 20 nodes for a single testcase on a github runner.